impedance/validation.py

```diff
@@ -80,8 +80,8 @@ def linKK(f, Z, c=0.85, max_M=50):
     def get_tc_distribution(f, M):
         """ Returns the distribution of time constants for the linKK method """
 
-        t_max = 1/np.min(f)
-        t_min = 1/np.max(f)
+        t_max = 1/(2 * np.pi * np.min(f))
```

Comment:
The units here seem weird to me. I agree that it's what they have in their paper, and it definitely works/matches better. Just seems like the radians are disappearing? Or the units of tau aren't seconds? 😕

Reply:
Yeah it's weird. I think it comes from the radians being assumed in angular frequencies and cycles (or waveforms) being assumed in linear frequency. Then 2 π is just the conversion of radians to cycles.